### ISSUE TITLE
Can't download Model

### ISSUE THREAD

**Issue by BerryGoesNomad** [TheLastBen/fast-stable-diffusion#1785]
please help, i clean up my Google drive and accidently deleted Fast-Dreambooth folder, and then when i launch fast dreambooth like before , can't download Model
![Screenshot_458](https://user-images.githubusercontent.com/101393609/226085162-fd90e4fe-374c-497d-8ac8-815753b1ea2d.png)

**Comment by kieranxsomer** [comment]
Wouldn't just running a reinstall work?

**Comment by TheLastBen** [comment]
it is downloading from what I see in the screenshot

**Comment by BerryGoesNomad** [comment]
> Wouldn't just running a reinstall work?

i tried but it not work

**Comment by BerryGoesNomad** [comment]
> it is downloading from what I see in the screenshot

**Comment by BerryGoesNomad** [comment]
> it is downloading from what I see in the screenshot

yes but, the "large alloc" thing keep spawning, and then "DONE" massage appear
when i start the Dreambooth it says "no model......." with red text
